core/src/main/java/com/linecorp/armeria/client/retry/RetryStrategy.java

```diff
@@ -89,20 +85,10 @@
     /**
      * Returns {@link CompletableFuture} with {@link Optional} that contains {@link Backoff} which will be
      * used for retry. If the condition does not match, this needs to return {@link Optional#EMPTY}
```

Comment:
Not directly related with this PR, but isn't it `{@link Optional#empty()}`?

Reply:
Oops~!! Thank you~!